ngraph_bridge/ngraph_builder.cc

```diff
@@ -92,15 +92,28 @@ static void SaveNgOp(Builder::OpMap& ng_op_map, const std::string& op_name,
   ng_op_map[op_name].push_back(output_node);
 }
 
-template <class TOpType, class... TArg>
-std::shared_ptr<TOpType> ConstructNgNode(const std::string& op_name,
-                                         TArg&&... Args) {
-  auto ng_node = std::make_shared<TOpType>(std::forward<TArg>(Args)...);
+void SetTracingInfo(const std::string& op_name,
```

Comment:
You may put this in utils so both builder and conversions can use it.

Reply:
Put it in `ngraph_builder.h`